tools/eslint-rules/prefer-primordials.js

```diff
@@ -23,10 +23,7 @@ function isTarget(map, varName) {
 }
 
 function isIgnored(map, varName, propName) {
-  if (!map.has(varName) || !map.get(varName).has(propName)) {
-    return false;
-  }
-  return map.get(varName).get(propName).ignored;
+  return map.get(varName)?.get(propName)?.ignored ?? false;
```

Comment:
To use optional chaining, we need to fix the syntax error of the build.
https://ci.nodejs.org/job/node-test-linter/37662/console

```
20:13:32 Running JS linter...
20:13:33 
20:13:33 Oops! Something went wrong! :(
20:13:33 
20:13:33 ESLint: 7.12.1
20:13:33 
20:13:33 /home/iojs/build/workspace/node-test-linter@2/tools/eslint-rules/prefer-primordials.js:26
20:13:33   return map.get(varName)?.get(propName)?.ignored ?? false;
20:13:33                           ^
20:13:33 
20:13:33 SyntaxError: Unexpected token '.'
20:13:33     at new Script (vm.js:88:7)
...
```

Reply:
@nodejs/ci-config-travis-maintainers is it possible to use Node.js 14 for the linter CI?

Reply:
ping @nodejs/ci-config-travis-maintainers

Reply:
ci.nodejs.org is Jenkins, not Travis. So if it's a job on ci.nodejs.org you're asking about, I think you'd want to ping `@nodejs/jenkins-admins`. (Not pinging myself in case I'm misunderstanding something here and this really is about the Travis configs in https://github.com/nodejs/ci-config-travis or something like that.)

Reply:
This is definitely about the configuration used to run the `node‑test‑linter` **Jenkins CI** job, so it needs mentioning <a class="team-mention js-team-mention" href="https://github.com/orgs/nodejs/teams/jenkins-admins"><b>@nodejs/jenkins‑admins</b></a>. (I can’t do it as I’m not a member of @nodejs)

Reply:
Thanks @Trott and @ExE-Boss, you're right. Sorry for having sent unsolicited pings to folks, I should have known better.

@nodejs/jenkins-admins do you know if we can use a more modern version of Node.js on the `node-test-linter` job?

Reply:
Request was made in https://github.com/nodejs/build/issues/2504.